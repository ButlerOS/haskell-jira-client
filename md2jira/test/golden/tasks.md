# epic
## story

- [ ] a task
with left over

- [ ] a task
  with right

> with quote?

- [ ] a task
> with comment

- [ ] a task
  > with comment
