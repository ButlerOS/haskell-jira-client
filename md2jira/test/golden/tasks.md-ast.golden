Document
    { intro = []
    , epics =
        [ Epic
            { mJira = Nothing
            , title = "epic"
            , description = []
            , stories =
                [ Story
                    { mJira = Nothing
                    , title = "story"
                    , description =
                        [ BulletList
                            [
                                [ Para
                                    [ Str "☐"
                                    , Space
                                    , Str "a"
                                    , Space
                                    , Str "task"
                                    , SoftBreak
                                    , Str "with"
                                    , Space
                                    , Str "left"
                                    , Space
                                    , Str "over"
                                    ]
                                ]
                            ,
                                [ Para
                                    [ Str "☐"
                                    , Space
                                    , Str "a"
                                    , Space
                                    , Str "task"
                                    , SoftBreak
                                    , Str "with"
                                    , Space
                                    , Str "right"
                                    ]
                                ]
                            ]
                        , BlockQuote
                            [ Para
                                [ Str "with"
                                , Space
                                , Str "quote?"
                                ]
                            ]
                        , BulletList
                            [
                                [ Plain
                                    [ Str "☐"
                                    , Space
                                    , Str "a"
                                    , Space
                                    , Str "task"
                                    ]
                                ]
                            ]
                        , BlockQuote
                            [ Para
                                [ Str "with"
                                , Space
                                , Str "comment"
                                ]
                            ]
                        , BulletList
                            [
                                [ Plain
                                    [ Str "☐"
                                    , Space
                                    , Str "a"
                                    , Space
                                    , Str "task"
                                    ]
                                , BlockQuote
                                    [ Para
                                        [ Str "with"
                                        , Space
                                        , Str "comment"
                                        ]
                                    ]
                                ]
                            ]
                        ]
                    , tasks =
                        [ Task
                            { status = Open
                            , title = "a task"
                            , assigned = []
                            , description =
                                [ Plain
                                    [ Str "with"
                                    , Space
                                    , Str "left"
                                    , Space
                                    , Str "over"
                                    ]
                                ]
                            }
                        , Task
                            { status = Open
                            , title = "a task"
                            , assigned = []
                            , description =
                                [ Plain
                                    [ Str "with"
                                    , Space
                                    , Str "right"
                                    ]
                                ]
                            }
                        , Task
                            { status = Open
                            , title = "a task"
                            , assigned = []
                            , description = []
                            }
                        , Task
                            { status = Open
                            , title = "a task"
                            , assigned = []
                            , description = []
                            }
                        ]
                    , mScore = Nothing
                    , updated = Nothing
                    }
                ]
            }
        ]
    }